packages/asset-swapper/src/utils/quote_requestor.ts

```diff
@@ -1,24 +1,20 @@
 import { schemas, SchemaValidator } from '@0x/json-schemas';
 import { assetDataUtils, orderCalculationUtils, SignedOrder } from '@0x/order-utils';
+import { FirmQuote, IndicativeQuote } from '@0x/quote-server';
 import { ERC20AssetData } from '@0x/types';
 import { BigNumber, logUtils } from '@0x/utils';
 import Axios, { AxiosResponse } from 'axios';
 
 import { constants } from '../constants';
 import { MarketOperation, RfqtMakerAssetOfferings, RfqtRequestOpts } from '../types';
 
+export { SignedOrder } from '@0x/order-utils';
```

Comment:
Do we need to export these here?

Reply:
My logic was that a module should export all types that its clients will need to reference in order to use that module's API.  We use both of these types in the interfaces for `QuoteRequestor`'s public methods `requestRfqtFirmQuotesAsync()` and `requestRfqtIndicativeQuotesAsync()`.

Reply:
This feels a bit strange to me considering we already export SignedOrder in `asset-swapper`'s index.ts for any consumer of this package: https://github.com/0xProject/0x-monorepo/blob/b9984b6df4555f4d4417d0589dff2f4214ad76a6/packages/asset-swapper/src/index.ts#L15

I'd prefer to remove this